metrics/cli/docs/features/doctor_output_improvements/diagrams/doctor_output_improvements_class_diagram.puml

```diff
@@ -0,0 +1,148 @@
+@startuml doctor_output_improvements_class_diagram
+
+package cli.lib as cli_lib {
+    package cli {
+        package command {
+            class DoctorCommand {
+                + name: String
+                + doctorFactory: DoctorFactory
+                + validationResultPrinter: ValidationResultPrinter
+
+                + run(): Future<void>
+            }
+        }
+
+        package doctor {
+            package factory as doctor_factory {
+                class DoctorFactory {
+                    - _serviceFactory: ServicesFactory
+
+                    + create(): Doctor
+                }
+            }
+
+            class Doctor {
+                - _coolService: CoolService
+                + validationResultBuilder: ValidationResultBuilder
+
+                + checkVersions(): Future<ValidationResult>
+            }
+        }
+    }
+
+    package services {
+        package common as services_common {
+            interface InfoService {
```

Comment:
:readable:
Maybe there is no sense in displaying this interface on the diagram since it does not add any useful context here? What do you think?